src/app/pages/dashboard/widgets/system/widget-sys-info-active/widget-sys-info-active.component.html

```diff
@@ -20,7 +20,7 @@
         @if (systemInfo()?.system_serial && isIxHardware()) {
           <div class="hostname">
             @if (isLoaded()) {
-              <span>
+              <span [matTooltip]="systemInfo()?.hostname">
```

Comment:
That's incorrect, it's expected to have that result same as on sidenav.
<img width="240" alt="Screenshot 2025-03-15 at 10 16 49" src="https://github.com/user-attachments/assets/27f232ae-6535-48d9-a353-ce2b793ac4a2" />

You should have: 

**${HOSTNAME}** (actual value) --> and once you hover it you should have a tooltip "Hostname: ${hostnameValue}"